src/pages/bylaws.js

```diff
@@ -389,7 +389,9 @@ const Bylaws = () => {
         <p>
           {' '}
           A vacancy in any office may be filled for the unexpired portion of the
-          term by a majority vote of the Board at any meeting.
+          term by a majority vote of the Board at any meeting. Each member of the
```

Comment:
As discussed at the ISC meeting, this looks good to the Council, changing "majority" -> "plurality".

Reply:
Done in 58861ae.